terraform/datadog/concourse.tf

```diff
@@ -0,0 +1,90 @@
+resource "datadog_monitor" "concourse-load" {
+  name               = "${format("%s concourse load", var.env)}"
+  type               = "query alert"
+  message            = "${format("Concourse load is too high: {{value}}. Check VM health. @govpaas-alerting-%s@digital.cabinet-office.gov.uk", var.aws_account)}"
+  escalation_message = "Concourse load still too high: {{value}}."
+  notify_no_data     = false
+  query              = "${format("avg(last_5m):avg:system.load.5{bosh-job:concourse,bosh-deployment:%s} > 200", var.env)}"
+
+  thresholds {
+    warning  = "150.0"
+    critical = "200.0"
+  }
+
+  require_full_window = true
+
+  tags {
+    "deployment" = "${var.env}"
+    "service"    = "${var.env}_monitors"
+    "job"        = "concourse"
+  }
+}
+
+resource "datadog_monitor" "continuous-smoketests" {
+  name               = "${format("%s continuous smoketests runtime", var.env)}"
```

Comment:
May I suggest we add the work "concourse" on this description. That way is easier to find the monitor.